tests/rptest/clients/kcl.py

```diff
@@ -275,14 +275,9 @@ def alter_partition_reassignments(self,
 
         lines = self._cmd(cmd).splitlines()
         self._redpanda.logger.debug(lines)
-        ok_re = re.compile(
-            r"^(?P<topic>[a-z\-]+?) +(?P<partition>[0-9]+?) +OK$")
         no_broker_re = re.compile(
             r"^(?P<topic>[a-z\-]+?) +(?P<partition>[0-9]+?) +BROKER_NOT_AVAILABLE.*$"
         )
-        bad_rep_factor_re = re.compile(
```

Comment:
this error condition is removed, but it should still be a valid error condition returned by the broker